deepchecks/vision/utils/label_prediction_properties.py

```diff
@@ -0,0 +1,100 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module containing measurements for labels and predictions."""
+from typing import List
+from itertools import chain
+
+import torch
+
+from deepchecks.core.errors import DeepchecksValueError
+
+
+# Labels
+
+def _get_bbox_area(labels: List[torch.Tensor]) -> List[int]:
+    """Return a list containing the area of bboxes in batch."""
+    areas = [(label.reshape((-1, 5))[:, 4] * label.reshape((-1, 5))[:, 3]).tolist()
+             for label in labels]
```

Comment:
Is this still the optimal way of getting the bbox area? Didn't we discuss a bbox class or something?

Reply:
surely I haven't discussed that, maybe @noamzbr